src/App.tsx

```diff
@@ -1,7 +1,29 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+export class App extends Component {
+  state = {
+    pressedKey: 'Nothing was pressed yet',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.getPressedKey);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.getPressedKey);
+  }
+
+  getPressedKey = (event: KeyboardEvent) => {
+    this.setState({ pressedKey: `Nothing was pressed yet [${event.key}]` });
```

Comment:
what's the logic? when I press some key, I see the message "Nothing was pressed yet [myPressedKey]"